CHANGELOG.rst

```diff
@@ -1,10 +1,11 @@
 CHANGELOG
 =========
 
-2.9.4.rc1 (2016-05-16)
-----------------------
+3.0.0 (2016-05-26)
+------------------
 
-* Adds support for the django CMS's ability to better handle plugin creation
+* Adds django CMS 3.3 support
```

Comment:
Very small thing. We've adopted the standard of using a past-tense verb at the start of each changelog entry, so this would be: `added support for django CMS 3.3`.